controllers/projects-controller.js

```diff
@@ -0,0 +1,70 @@
+let express = require('express');
+let Project = require('../models/projects-model');
+
+getAllProjects = (request, response) => {
+    // query db; if no errors, send all books
+    let query = Project.find();
+    query.exec((error, projects) => {
+      if (error) {
+        response.send(error);
+      } else {
+        response.json(projects);
+      }
+      
+    });
+}
+
+getProjectById = (request, response) => {
+  let id = request.params.id;
+  let query = Project.find({ _id: id });
+  query.exec((error,project) => {
+    if (error) { 
+      response.send(error);
+    } else {
+      response.json(project);
+    }
+  });
+}
+
+addProject = (request, response) => {
+  let newProject = new Project({
```

Comment:
@andrewoconnell89 
Andy,
All these fields are currently not visible in the UI when we are adding a new project. will it be done as a part of a future Pull request?